package.json

```diff
@@ -36,9 +36,9 @@
     "generate.icons": "yarn run fmt.svg && node ./scripts/generate-icons.js ",
     "prebuild": "yarn prep",
     "build": "tsc --noEmit && yarn rollup",
-    "wds": "wds --app-index dev/index.html --watch --node-resolve",
+    "wds": "wds --app-index dev/index.html --open dev/ --watch --node-resolve",
     "preserve": "yarn prep",
-    "serve": "concurrently -c \"cyan,blue\" \"yarn:rollup.watch\" \"yarn:wds\" --kill-others",
+    "serve": "concurrently -c \"cyan,blue\" \"yarn:wds\" \"yarn:rollup.watch\" --kill-others",
```

Comment:
I'm going to merge this, but I still think long term, concurrently is causing the issues with various race conditions. My hunch is that this is a "lucky" fix.